cookies/templates/resources.html

```diff
@@ -61,6 +61,7 @@ <h4 class="panel-title">
         <a class="list-group-item"
             href="{{ resource.get_absolute_url }}">
             <span class="label label-primary">{{ resource.entity_type }}</span> {{ resource.name }}
+            <br><small style="color:lightslategray">{{ resource.created_by }}</small>
```

Comment:
@sanjanavsdvn Cool! We are using the bootstrap framework, so if possible I'd suggest using existing classes from that package. Take a look at http://getbootstrap.com/css/#helper-classes-colors -- perhaps ``text-muted`` or ``text-info`` ? 

If you can't find something that you like, you can always write a new CSS class in [static/cookies/css/base.css](https://github.com/diging/jars/blob/master/cookies/static/cookies/css/base.css) -- use names similar to bootstrap, e.g. ``text-cool``, ``text-awesome``, etc.